pinot-spi/src/main/java/org/apache/pinot/spi/data/Schema.java

```diff
@@ -688,13 +688,18 @@ public void updateBooleanFieldsIfNeeded(Schema oldSchema) {
     }
   }
 
+
+  public boolean isBackwardCompatibleWith(Schema oldSchema) {
+    return isBackwardCompatibleWith(oldSchema, false);
+  }
+
   /**
    * Check whether the current schema is backward compatible with oldSchema.
    * Backward compatibility requires all columns and fieldSpec in oldSchema should be retained.
    *
    * @param oldSchema old schema
    */
-  public boolean isBackwardCompatibleWith(Schema oldSchema) {
+  public boolean isBackwardCompatibleWith(Schema oldSchema, boolean skipTimeColumn) {
```

Comment:
Can you please rebase to the latest master? We already fixed this and allow time format change for time